src/Hl7.Fhir.Specification/Specification/Source/DirectorySource.cs

```diff
@@ -42,14 +40,9 @@ namespace Hl7.Fhir.Specification.Source
     [DebuggerDisplay(@"\{{DebuggerDisplay,nq}}")]
     public class DirectorySource : ISummarySource, IConformanceSource, IArtifactSource
     {
-        // netstandard has no CurrentCultureIgnoreCase comparer
-#if DOTNETFW
         private static readonly StringComparer ExtensionComparer = StringComparer.InvariantCultureIgnoreCase;
```

Comment:
Waarom is de #if hier weg? Er was blijkbaar een verschil tussen InvariantCultureIgnoreCase en OrdinalCultureIgnoreCase, dat valt nu weg.

Reply:
HL7.Fhir.Specification targets only net45 and netstandard2.0. So no netstandard1.1 anymore. That's why this conditional directive could be deleted.